packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -121,6 +130,7 @@ function ButtonWithRef(
       role="menuitem"
       $isIconMenu={isIconMenu}
       $isHorizontal={isHorizontal}
+      $forcePadding={forcePadding}
```

Comment:
From recent discussions: These props shouldn't need being transient, see also https://xwp.slack.com/archives/CCCAUAH6F/p1643891495809269 and https://github.com/GoogleForCreators/web-stories-wp/issues/10468 and https://github.com/GoogleForCreators/web-stories-wp/issues/6669#issuecomment-793531492

Reply:
Excellent points. I have reworked it in c9c0c43dba.